openslides/motions/static/js/motions/pdf.js

```diff
@@ -657,9 +659,22 @@ angular.module('OpenSlidesApp.motions.pdf', ['OpenSlidesApp.core.pdf'])
                     image_sources = image_sources.concat(tmp_image_sources);
                 });
 
-                //post-request to convert the images. Async.
-                $http.post('/core/encode_media/', JSON.stringify(image_sources)).then(function (success) {
-                    var converter = PdfMakeConverter.createInstance(success.data.images);
+                var image_map = [];
+                _.forEach(image_sources, function (image_source) {
+                    image_map[image_source] = PDFLayout.imageURLtoBase64(image_source);
```

Comment:
This code was not easy to understand for me because we have an empty array `image_map` and set some properties of this array (but length of array remains 0, so the content is somehow hidden). Later you iterate over these properties ....

To make this code better maintainable you can use something like
`image_map.push({image_source: PDFLayout.imageURLtoBase64(image_source)});`

But the code is inside motions/pdf.js so you and @emanuelschuetze have to maintain it ... your choice. :-)